portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,35 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Facundo's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="background-color:Gainsboro;">
```

Comment:
No need to fix right now but I recommend all styling be done in a separate CSS sheet. It's also probably easier and more consistent to use hex or rgba values for colors.